File: Beecrowd/Beginner/1435 - 1985/1828 - Bazinga!/Python/1828_Bazinga!.py
test_cases = int(input())

for i in range(1, test_cases + 1):
    sheldon, raj = input().split()

    if sheldon == "tesoura" and raj == "papel":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "papel" and raj == "pedra":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "pedra" and raj == "lagarto":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "lagarto" and raj == "Spock":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "Spock" and raj == "tesoura":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "tesoura" and raj == "lagarto":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "lagarto" and raj == "papel":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "papel" and raj == "Spock":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "Spock" and raj == "pedra":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == "pedra" and raj == "tesoura":
        print(f"Caso #{i}: Bazinga!")
    elif sheldon == raj:
        print(f"Caso #{i}: De novo!")
    else:
        print(f"Caso #{i}: Raj trapaceou!")

File: Beecrowd/Beginner/1435 - 1985/1828 - Bazinga!/C/1828_Bazinga!.c
#include <stdio.h>
#include <string.h>

int main() {
    int testCases;
    scanf("%d", &testCases);

    for (int i = 1; i <= testCases; i++) {
        char sheldon[10], raj[10];
        scanf("%s %s", sheldon, raj);

        if (strcmp(sheldon, "tesoura") == 0 && strcmp(raj, "papel") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "papel") == 0 && strcmp(raj, "pedra") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "pedra") == 0 && strcmp(raj, "lagarto") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "lagarto") == 0 && strcmp(raj, "Spock") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "Spock") == 0 && strcmp(raj, "tesoura") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "tesoura") == 0 && strcmp(raj, "lagarto") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "lagarto") == 0 && strcmp(raj, "papel") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "papel") == 0 && strcmp(raj, "Spock") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "Spock") == 0 && strcmp(raj, "pedra") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, "pedra") == 0 && strcmp(raj, "tesoura") == 0)
            printf("Caso #%d: Bazinga!\n", i);
        else if (strcmp(sheldon, raj) == 0)
            printf("Caso #%d: De novo!\n", i);
        else printf("Caso #%d: Raj trapaceou!\n", i);
    }

    return 0;
}
